internal/app/carrier/service_test.go

```diff
@@ -0,0 +1,58 @@
+package carrier
+
+import (
+	"context"
+	"errors"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/assert"
+	"parcel-service/internal/app/model"
+	"parcel-service/internal/app/service/mocks"
+	"testing"
+)
+
+func TestService_AssignCarrierToParcel(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	testCases := []struct {
+		desc     string
+		payload  model.CarrierRequest
+		mockRepo func() *mocks.MockCarrierParcelAcceptRepository
+		expErr   error
+	}{
+		{
+			desc: "should return success",
+			payload: model.CarrierRequest{
```

Comment:
Same as previous, use common variable for avoiding redundancy .

Reply:
removed